packages/prisma/schema.prisma

```diff
@@ -570,60 +570,60 @@ enum BookingStatus {
 }
 
 model Booking {
-  id                    Int                        @id @default(autoincrement())
-  uid                   String                     @unique
+  id                           Int                            @id @default(autoincrement())
+  uid                          String                         @unique
   // (optional) UID based on slot start/end time & email against duplicates
-  idempotencyKey        String?                    @unique
-  user                  User?                      @relation(fields: [userId], references: [id], onDelete: Cascade)
-  userId                Int?
+  idempotencyKey               String?                        @unique
+  user                         User?                          @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId                       Int?
   // User's email at the time of booking
```

Comment:
Remove all the unnecessary file changes

Reply:
Hey @MuhammadAimanSulaiman , can you also sign CLA.